pkg/rancher-desktop/backend/lima.ts

```diff
@@ -305,7 +305,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
       this.emit('progress');
     });
 
-    if (!(process.env.NODE_ENV ?? '').includes('test')) {
+    if (!(process.env.RD_TEST ?? '').includes('e2e')) {
```

Comment:
I was going to say things about `NODE_ENV == test` but I don't think this code is relevant for unit tests.

Reply:
As far as I can tell, this isn't used for unit tests. Besides, we should probably stub out anything that needs it for unit tests instead of relying on a condition like that.